src/components/DownloadPanel/DownloadPanel.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <form action=""
+    class="download-panel">
+    <h3 class="download-panel__title">Export icon</h3>
+    <div class="download-panel__parameters">
+      <fieldset class="download-panel__parameter">
+        <legend class="download-panel__parameter-name">Color</legend>
+        <div class="input-box-row">
+          <input type="radio"
+            name="color"
+            value="dark-blue"
+            id="dark-blue"
+            class="input-box"
+            v-model="color">
+          <label for="dark-blue"
+            class="input-box-label">
+            Dark blue
+          </label>
+        </div>
+        <div class="input-box-row">
+          <input type="radio"
+            name="color"
+            value="white"
+            id="white"
+            class="input-box"
+            v-model="color">
+          <label for="white"
+            class="input-box-label">
+            White
+          </label>
+        </div>
+      </fieldset>
+
+      <div class="download-panel__parameter">
+        <span class="download-panel__parameter-name">Size</span>
+        <select type="select"
+          id="size"
+          class="input-select download-panel__select">
```

Comment:
Vues JS has something more elegant to do this, no ? [ici](https://vuejs.org/v2/guide/forms.html#Select)